static/src/javascripts/projects/commercial/modules/liveblog-adverts.ts

```diff
@@ -94,23 +94,37 @@ const getSlotName = (isMobile: boolean, slotCounter: number): string => {
 	return `inline${slotCounter + 1}`;
 };
 
-const insertAds: SpacefinderWriter = async (paras) => {
+const insertAdAtPara = (para: Node): Promise<void> => {
```

Comment:
Take it or leave this one, but I'm wondering if while you're here it makes sense to change `para` -> `block`, since its liveblogs?

Reply:
or `winningBlock`

Reply:
Oops I missed this sorry! @Jakeii lets remember to do this when we remove the temporary container class